### PR TITLE
[20023] Bump version to v2.10.3

### PR DESCRIPTION
**Pull request by JesusPoderoso**

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/141/)

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/153/)

File: docs/notes/previous_versions/v2.10.1.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Version 2.10.1
 ^^^^^^^^^^^^^^
-This minor release includes the following **improvements**:
+This patch release includes the following **improvements**:
```
Comment (EduPonz) [inline docs/notes/previous_versions/v2.10.1.rst:3]:
```suggestion

This patch release includes the following **improvements**:
```

File: docs/notes/previous_versions/v2.10.2.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Version 2.10.2
+^^^^^^^^^^^^^^
+This patch release includes the following **improvements**:
```
Comment (EduPonz) [inline docs/notes/previous_versions/v2.10.2.rst:3]:
```suggestion

This patch release includes the following **improvements**:
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/154/)

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/164/)

**Review by MiguelCompany — APPROVED** [review]

**Pull request merged**